Assets/Scripts/SlideSwitcher.cs

```diff
@@ -1,8 +1,12 @@
+using System.Collections;
 using UnityEngine;
 
 public class SlideSwitcher : MonoBehaviour
 {
 	[SerializeField] SlideHolder slideHolder;
+
+	bool onCooldown;
```

Comment:
By the C# convention, we usually add an underscore in front of the private variables' names (so here it would be _onCooldown). It's quite useful since you know later in the code which variables are private without having to check in their declaration :)